lib/parse.js

```diff
@@ -2,7 +2,10 @@
 
 const {URL} = require('url');
 
-const semverRegex = require('semver-regex');
+/* Inlining semver-regex v2.0.0 here instead of unreliably pinning it in package.json.
+ * This is the last version without look-behind so it can run in older browsers supported by Stylus.
+ * https://github.com/sindresorhus/semver-regex/releases/tag/v2.0.0 */
+const RE_SEMVER = /\bv?(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)(?:-[\da-z-]+(?:\.[\da-z-]+)*)?(?:\+[\da-z-]+(?:\.[\da-z-]+)*)?\b/ig;
```

Comment:
```suggestion
const RE_SEMVER = /\bv?(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)(?:-[\da-z-]+(?:\.[\da-z-]+)*)?(?:\+[\da-z-]+(?:\.[\da-z-]+)*)?\b/ig; // eslint-disable-line
```

Reply:
@silverwind it didn't help, see the details...